java/src/main/java/org/apache/arrow/gandiva/expression/TreeBuilder.java

```diff
@@ -46,6 +46,14 @@ public static TreeNode makeLiteral(Long longConstant) {
     return new LongNode(longConstant);
   }
 
+  public static TreeNode makeStringLiteral(String stringConstant) {
```

Comment:
naming convention different from existing functions..

Reply:
For the cpp API, I had to keep different names because both accept the same argument (std::string). For Java API, I kept the names to be match that of the cpp API.

Reply:
I think Praveen's point is that the Java API can be makeLiteral(String) and makeLiteral(byte[]) like the other makeLiteral APIs.

Reply:
understood, Vivek. we need to choose between "consistency between java and cpp APIs" and "consistency within java APIs". I chose the former.